api-docs/get-interactions-hours.md

```diff
@@ -0,0 +1,69 @@
+# Number of Interactions per Hour Range.
+
+Returns the total number of clients who have had at least that many client interaction hours in the given time period, along with the list of clients that match the specified criteria.
+
+## Request
+
+_The request should accept query parameters indicating a time range, number of client interaction hours, and current page._
+
+```http
+GET /api/interactions?start=2019-05-06T0&end=2019-05-08T0&minInteractionMinutes=21600&page=1
+```
+
+Notes:
+
+- Result is limited to 100 rows ordered by lastName and then firstName
+- Values for the query should be URL encoded
+- The 'page' query parameter defaults to 1
+- The 'start' parameter denotes the lower class limit (date and time)
+- The 'end' parameter denotes the upper class limit (date and time)
+- The 'minInteractionSeconds' parameter specifies the least number of hours the query will filter
+- reportDefaultParameters explains values for parameters left blank or empty
+
+## Response
+
+```json
+{
+  "clients": [
+    {
+      "id": 1,
+      "firstName": "John",
+      "lastName": "Doe",
+      "fullName": "John Doe",
+      "gender": "male",
+      "birthday": "1969-08-15",
+      "totalDuration": "06:00:00"
+    },
+    {
+      "id": 2,
+      "firstName": "Jane",
+      "lastName": "Doe",
+      "fullName": "Jane Doe",
+      "gender": "female",
+      "birthday": "1969-08-16",
+      "totalDuration": "06:15:00"
+    },
+    {
+      "id": 3,
+      "firstName": "Jill",
+      "lastName": "Doe",
+      "fullName": "Jill Doe",
+      "gender": "female",
+      "birthday": "1969-08-17",
+      "totalDuration": "06:30:00"
+    }
+  ],
+  "pagination": {
+    "numClients": 3,
+    "currentPage": 1,
+    "pageSize": 100,
+    "numPages": 1
+  },
+  "reportDefaultParameters": {
+    "start": "[TODAY]", // it will default to currentday
```

Comment:
You can't put comments into json 😭. Doing so causes this:

![image](https://user-images.githubusercontent.com/5524384/69106558-ff38b780-0a2b-11ea-8927-1c472977c587.png)

```suggestion
    "start": "[TODAY]",
```

Reply:
My bad lol. I was in a javascript mindset😆, I'll get those removed right away.